tutorial/tutorial.md

```diff
@@ -0,0 +1,245 @@
+linter tool.
+
+Making your project REUSE compliant can be done in three simple steps:
+
+1. Choose and provide licenses
+2. Add copyright and license information to each file
+3. Confirm REUSE compliance
+
+For the purpose of this tutorial, we will assume that the directory of your
+project looks like this:
+
+```
+project/
+├── img/
+│   ├── cat.jpg
+│   └── dog.jpg
+├── src/
+│   └── main.c
+├── .gitignore
+├── Makefile
+└── README.md
+```
+
+If you would like to reproduce the steps in this tutorial on your own
+computer, you can clone the [example
+repository](https://github.com/fsfe/reuse-example). The branch `noncompliant`
+matches the structure above, while the `master` branch is the successful
+result of this repository.
+
+## 1. Choose and provide licenses
+
+The first thing you need to do is to [choose a
+license](https://choosealicense.com/). For this tutorial, we assume that you
+chose the GNU General Public Licence v3.0 or any later version (GPL). More
+than simply choosing a license, you need to put the license in your project
+directory.
+
+You find your license in the [SPDX License List](https://spdx.org/licenses/).
+SPDX is an open standard for communicating license and copyright information.
+Each license is uniquely identified by a shortform [SPDX
+identifier](https://spdx.org/ids). The SPDX identifier for your chosen license
+is GPL-3.0-or-later.
+
+<!-- TODO: Use tool instead -->
+
+You create a `LICENSES` directory in your project root which will contain all
+the licenses that you use in your project. You then download your license
+from the
+[license-list-data](https://github.com/spdx/license-list-data/tree/master/text)
+repository and put it in the `LICENSES` directory.
+
+## 2. Add copyright and license information to each file
+
+Now that you have a license, you need to indicate in the relevant files that
+these files fall under that license. You edit the comment header of
+`src/main.c` as such:
+
+```
+/*
+ * SPDX-Copyright: 2019 Jane Doe <jane@example.com>
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+```
+
+<!-- FIXME: Fix links -->
+
+The `SPDX-Copyright` tag records the publication years and copyright holder
+of the contents of the file. You can read more about [which publication years
+to use]() and [what copyright holders are]() in the FAQ.
+
+The `SPDX-License-Identifier` tag is followed by a [valid SPDX License
+Expression](https://spdx.org/specifications), typically just the SPDX
+Identifier of the license.
+
+<!-- FIXME: Fix links -->
+
+Each file must always contain these two tags in the header. You are allowed
+to [use the tags multiple times if you have multiple copyright holders or
+licenses]().
+
+In the example project, you also edit `Makefile` and `README.md` using this
+header information, but of course with corresponding comment syntax.
+
+### Binary and uncommentable files
+
+You also want to license your image files under GPL-3.0-or-later.
```

Comment:
I think you're not reading the intent of the section. It has the following intent:

- A non-plaintext file cannot contain a header. An image is a very obvious example.

- We _expressly_ license these wrongly so that we can fix them in a later paragraph.

- Adding the complexity of "licensing images under the GPL is a bad idea" makes this tutorial longer than it needs to be, and one of the express goals of the tutorial is to be as short as possible.

- CC0 images were chosen so that we can safely lie about their licensing (while still referring to the authors if people are so inclined). But @mxmehl  is proposing to fix this with his own images, which also works.